action.yml

```
@@ -0,0 +1,51 @@
+name: "syncer"
+author: "GOG Nebula"
+description: "Sync galaxy integration forks with upstream"
+inputs:
+  upstream:
+    description: 'Upstream repo of the fork. (eg. Codertocat/Hello-World)'
+    required: true
+  reviewers:
+    description: 'A comma or newline separated list of reviewers (GitHub usernames) to request a review from.'
+runs:
+  using: 'composite'
+  steps:
+    - name: Checkout base
+      uses: actions/checkout@v3
+    - name: Get base version
+      id: base_version
+      uses: notiz-dev/github-action-json-property@release
+      with:
+        path: 'src/manifest.json'
+        prop_path: 'version'
+    - run: echo Got base version: ${{steps.base_version.outputs.prop}}
+      shell: bash
+    - name: Checkout upstream
+      uses: actions/checkout@v3
+      with:
+        repository: ${{ inputs.upstream }}
+    - name: Get upstream version
+      id: upstream_version
+      uses: notiz-dev/github-action-json-property@release
+      with:
+        path: 'src/manifest.json'
+        prop_path: 'version'
+    - run: echo Got upstream version: ${{steps.upstream_version.outputs.prop}}
+      shell: bash
+    - name: Create pull request
+      id: cpr
+      if: ${{ steps.upstream_version.outputs.prop != steps.base_version.outputs.prop }}
+      uses: peter-evans/create-pull-request@v4
+      with:
+        base: ${{ github.repository }}
+        branch: pr/sync
```

maybe just "sync" or "autosync"?

---

yeah that works, I just arbitrarily chose something. Actually, I kind of like the `/` convention to give a clue as to what the branch is for. For example, `hotfix/appid` or `feature/ui`. Actually now that I think of it I can't think of uses for `pr/` so maybe `auto/sync`? In case we have another action that automatically creates a branch? Honestly tho, `autosync` is perfectly fine, feel free to make that change or if you have another alternative suggest it and I'll make it.
```suggestion
        branch: autosync
```